openmc/material.py

```diff
@@ -91,6 +92,13 @@ class Material(IDManagerMixin):
     fissionable_mass : float
         Mass of fissionable nuclides in the material in [g]. Requires that the
         :attr:`volume` attribute is set.
+    decay_photon_energy : openmc.stats.Univariate
+        Energy distribution of photons emitted from decay of unstable nuclides
+        within the material. The integral of this distribution is the total
+        intensity of the photon source in [decay/sec].
```

Comment:
Should we mention the requirements for this property? (chain file set in config and volume set)

Reply:
If they're not set, the user will get an exception telling them they need to be set, so I didn't think it was necessary to explicitly say so. At this point there are a few properties/methods like this that require volume information and we usually don't spell it out explicitly. If you feel strongly though, I can add it in the description.

Reply:
Good point. I think that should be enough.